### PR TITLE
[autoware_web_controller]add readme

### PR DESCRIPTION
**Pull request by tkimura4**
## Description

add readme of autoware_web_controller

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: common/util/web_controller/autoware_web_controller/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+| ------------------------------------------------------------------------------------------------------- | ---------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/path_change_approval` | `autoware_planning_msgs/msg/Approval`    | Send a approval signal for path change request such as lane change or obstacle avoidance |
+| `/autoware/engage`                                                                                      | `autoware_auto_system_msgs::msg::Engage` | Send a engage signal for Autoware                                                        |
+| `/vehicle/engage`                                                                                       | `autoware_auto_system_msgs::msg::Engage` | Send a engage signal for a vehicle                                                       |
```
Comment (kmiya) [inline common/util/web_controller/autoware_web_controller/README.md:25]:
```suggestion
| `/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/path_change_approval` | `autoware_planning_msgs/msg/Approval`    | Send an approval signal for path change request such as lane change or obstacle avoidance |
| `/autoware/engage`                                                                                      | `autoware_auto_system_msgs::msg::Engage` | Send an engage signal for Autoware                                                        |
| `/vehicle/engage`                                                                                       | `autoware_auto_system_msgs::msg::Engage` | Send an engage signal for a vehicle                                                       |
```
Nit: a -> an

File: common/util/web_controller/autoware_web_controller/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### Input
+
+| Name                                               | Type                                            | Description                  |
+| -------------------------------------------------- | ----------------------------------------------- | ---------------------------- |
+| `/control/current_gate_mode`                       | `autoware_control_msgs::msg::GateMode`          | Gate mode (AUTO or EXTERNAL) |
+| `/autoware/state`                                  | `autoware_auto_system_msgs::msg::AutowareState` | State of Autoware            |
+| `/autoware/engage`                                 | `autoware_auto_system_msgs::msg::Engage`        | Engage signal for Autoware   |
+| `/vehicle/engage`                                  | `autoware_auto_system_msgs::msg::Engage`        | Engage signal for a vehicle  |
+| `/planning/scenario_planning/max_velocity_default` | `autoware_planning_msgs/VelocityLimit`          | Max velocity of Autoware     |
```
Comment (taikitanaka3) [inline common/util/web_controller/autoware_web_controller/README.md:17]:
```suggestion
| `/planning/scenario_planning/max_velocity_default` | `autoware_planning_msgsmsg::VelocityLimit`          | Max velocity of Autoware     |
```

Comment (tkimura4) [reply common/util/web_controller/autoware_web_controller/README.md]: 7e3bf5a8

Comment (tkimura4) [reply common/util/web_controller/autoware_web_controller/README.md]: @taikitanaka3 
Done.

**Review by taikitanaka3 — APPROVED** [review]
LGTM

**Pull request merged**